File: backends/php-laravel/app/UseCases/Auth/Dtos/AuthUserDto.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\UseCases\Auth\Dtos;

use App\Support\Dtos\AbstractDto;

/**
 * 認証ユーザーDTOクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\UseCases\Auth\Dtos
 */
class AuthUserDto extends AbstractDto
{
    public ?int $id = null;
}

File: backends/java-spring/src/main/java/com/authorization/usecases/auth/dtos/AuthUserDto.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.usecases.auth.dtos;

import com.authorization.support.dtos.AbstractDto;
import lombok.Getter;
import lombok.Setter;

/**
 * 認証ユーザーDTOクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Getter
@Setter
public class AuthUserDto extends AbstractDto {

    private Long id;
}
